README.md

```diff
@@ -1 +1,12 @@
 # repository
+Repository is the repository to end all repositories. Builds in seconds. No dependancies. Every line of code extensively covered. This is the only repository you will ever need.
+
+## Installation
+It couldn't be easier.
+
+`git clone https://github.com/JDvorak/repository.git`
```

Comment:
Thanks for bringing this to my attention.
https://github.com/dlmather/repository/commit/8e6048c5f97f3c9971f6d9f6392ee5f2859e95d5